cf-key/cf-key.c

```diff
@@ -249,6 +252,10 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             KEY_PATH = optarg;
             break;
 
+        case 'T':
+            KEY_SIZE = atoi(optarg);
```

Comment:
You need some error checking here. @olehermanse what is the function you suggest here?

Reply:
`KEY_SIZE = StringToLongExitOnError(optarg);`